fluentd/configs.d/openshift/output-applications.conf

```diff
@@ -0,0 +1,5 @@
+<match **>
+   @type copy
+   @include output-es-config.conf
+   @include ../dynamic/es-copy-config.conf
```

Comment:
I think I'd argue against letting them install their own plugins, however I feel like we should provide them with the secure forward plugin.  That way they can forward their logs to any other fluentd instance they may want to use (to forward to any other log store)